homeassistant/components/tuya/binary_sensor.py

```diff
@@ -3,6 +3,7 @@
 from __future__ import annotations
 
 from dataclasses import dataclass
+import json
```

Comment:
For performance reasons, please use our helpers in `homeassistant.helpers.json` instead.

Reply:
Done 👍